src/common/POLFields/FieldMaterialType.js

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
-import { Field } from 'redux-form';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
 
-import { Select } from '@folio/stripes/components';
+import { FieldSelect } from '@folio/stripes-acq-components';
 
-import { EMPTY_OPTION } from '../../components/Utils/const';
 import normalizeEmptySelect from '../../components/Utils/normalizeEmptySelect';
 import { requiredMaterialType } from '../../components/Utils/Validate';
 
 const FieldMaterialType = ({ name, materialTypes, disabled, required }) => (
-  <Field
-    component={Select}
-    dataOptions={[EMPTY_OPTION, ...materialTypes]}
+  <FieldSelect
+    dataOptions={materialTypes}
     fullWidth
     label={<FormattedMessage id="ui-orders.poLine.materialType" />}
     name={name}
     normalize={normalizeEmptySelect}
+    required={required}
     validate={required && [requiredMaterialType]}
```

Comment:
does it make sense to replace `requiredMaterialType` with just `required`?

Reply:
`required && validateRequired` or something like this